packages/dx-react-grid/docs/reference/integrated-selection.md

```diff
@@ -14,7 +14,7 @@ A plugin that performs built-in selection.
 
 Name | Plugin | Type | Description
 -----|--------|------|------------
-selection | Getter | Set&lt;number &#124; string&gt; | The selected rows.
+selection | Getter | Array&lt;number &#124; string&gt; | The selected row identifiers.
```

Comment:
Maybe a better description is `The selected row IDs`? Because we already used `IDs` in our docs.